getfunction.c

```diff
@@ -53,15 +53,21 @@ unsigned int str_length(char *str)
 int print_integer(va_list valist, options opt)
 {
 	char *str;
+	char flag;
 	int bytes = 0;
-	(void) opt;
 
 	str = number_to_string(va_arg(valist, int));
 	if (str == NULL)
 	{
 		return (_putchar("(null)", str_length("(null)")));
 	}
-	bytes = _putchar(str, str_length(str));
+	if (opt.signp == 1)
```

Comment:
add issue to add this line depends of the type of convertion 
only works in integer?